### PR TITLE
Enable runsc/boot support on arm64.

### PR DESCRIPTION
**Pull request by xiaobo55x**
Signed-off-by: Haibo Xu <haibo.xu@arm.com>
Change-Id: I00beb9cc492e44c762ebaa3750201c63c1f7c2f3

### CODE REVIEW

**Comment by xiaobo55x** [comment]
Ref #63.

**Comment by xiaobo55x** [comment]
Hi @fvoznika, @prattmic,

Please help review this patch!

Thanks,
Haibo

File: runsc/boot/loader_amd64.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+// limitations under the License.
+
+// +build amd64
+
+// Package boot loads the kernel and runs a container.
+package boot
+
+import (
+	"gvisor.dev/gvisor/pkg/sentry/kernel"
+	slinux "gvisor.dev/gvisor/pkg/sentry/syscalls/linux"
```
Comment (fvoznika) [inline runsc/boot/loader_amd64.go]: nit: `slinux` is not needed

File: runsc/boot/loader_arm64.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+// limitations under the License.
+
+// +build arm64
+
+// Package boot loads the kernel and runs a container.
+package boot
+
+import (
+	"gvisor.dev/gvisor/pkg/sentry/kernel"
+	slinux "gvisor.dev/gvisor/pkg/sentry/syscalls/linux"
```
Comment (fvoznika) [inline runsc/boot/loader_arm64.go]: nit: `slinux` is not needed

File: runsc/boot/filter/config.go [hunk 3]
```diff
@@ -44,7 +40,7 @@ var allowedSyscalls = seccomp.SyscallRules{
 	},
 	syscall.SYS_CLOSE:         {},
 	syscall.SYS_DUP:           {},
-	syscall.SYS_DUP2:          {},
+	syscall.SYS_DUP3:          {},
```
Comment (fvoznika) [inline runsc/boot/filter/config.go]: Does this need to change? I don't see dup2 => dup3 in this change.

Comment (xiaobo55x) [reply runsc/boot/filter/config.go]: The change is committed in a previous commit(a25a976) from @lubinszARM. I think he may missed some changes in this file.

File: runsc/boot/loader_amd64.go (hunk 1 above)
Comment (xiaobo55x) [reply runsc/boot/loader_amd64.go]: Done!

File: runsc/boot/loader_arm64.go (hunk 2 above)
Comment (xiaobo55x) [reply runsc/boot/loader_arm64.go]: Done!

File: runsc/boot/filter/config.go (hunk 3 above)
Comment (xiaobo55x) [reply runsc/boot/filter/config.go]: Have collected all the related changes to this patch.

**Comment by xiaobo55x** [comment]
@fvoznika, thanks!

**Comment by xiaobo55x** [comment]
Hi @fvoznika,

It seems that this pull and #1177 were blocked by the  feedback/copybara check failure, could you help have a look? BTW, I saw this kind of error several times before, and  all of which were handled manfully by the reviewer. Is there any special attention we need to take when posting new patch to avoid this kind of check failure?

Thanks,
Haibo

**Comment by fvoznika** [comment]
Feedback/copybara steps normally fails because BUILD format and Golang
imports work slightly different with our internal tools and require manual
fixed ups. There is nothing you could have done in your end, thanks for
asking.

**Comment by xiaobo55x** [comment]
@fvoznika, thanks for the reply!

**Pull request merged**